base/redactor.go

```diff
@@ -32,6 +35,25 @@ func redact(args []interface{}) []interface{} {
 	return args
 }
 
+func (redactorSlice RedactorSlice) Redact() string {
+	tmp := []byte{}
+	for _, item := range redactorSlice {
+		tmp = append(tmp, []byte(item.Redact())...)
+		tmp = append(tmp, ' ')
+	}
+	return "[ " + string(tmp) + "]"
+}
+
+func buildSlice(valueOf reflect.Value, function func(interface{}) Redactor) RedactorSlice {
```

Comment:
Maybe a more specific name than `buildSlice` given this is in the base package. Could be confusing to see this name pop up in autocomplete.